util/homebrew/chapel-main.rb

```diff
@@ -0,0 +1,430 @@
+class Chapel < Formula
+  desc "Programming language for productive parallel computing at scale"
+  homepage "https://chapel-lang.org/"
+  url "https://github.com/chapel-lang/chapel/releases/download/1.25.1/chapel-1.25.1.tar.gz"
```

Comment:
I would expect this to be `chapel-release.rb` and the other to be `chapel-main` ? Anyway I am currently thinking it is best for this to be post-release; in that event, we can merge the Homebrew formula we end up with for 1.26 as chapel-release.rb.